CSharp/BatchExplorer/README.md

```diff
@@ -1,5 +1,7 @@
 ## Azure Batch Explorer
 
+**Note:** For an officially supported cross-platform client tool to browse Azure Batch resources see [Batch Labs](https://github.com/Azure/BatchLabs). Batch Explorer will continue to exist in this repository only as a sample and may not be updated to expose the latest features of Batch.
```

Comment:
instead of 'officially' should we say 'For a supported up-to-date cross-platform client tool'? 'Official' to me sounds more like something we're going to distribute.

Also, minor nitpick but you mention "Azure Batch" at first, and then say "Batch" later on. Maybe always use Azure Batch for consistency?